src/obu.c

```diff
@@ -1,6 +1,6 @@
 /*
- * Copyright © 2018, VideoLAN and dav1d authors
- * Copyright © 2018, Two Orioles, LLC
+ * Copyright (C) 2018, VideoLAN and dav1d authors
+ * Copyright (C) 2018, Two Orioles, LLC
```

Comment:
Is it possible to change these back to the copyright symbol? This is dav1d's license header. It would be better if we can have it in the original form.